src/test/java/org/jabref/logic/importer/fetcher/BvbFetcherTest.java

```diff
@@ -23,8 +22,30 @@
 class BvbFetcherTest {
 
     BvbFetcher fetcher = new BvbFetcher();
-    BibEntry bibEntryISBN0134685997;
-    BibEntry bibEntryISBN9783960886402;
+    BibEntry bibEntryISBN9783960886402 = new BibEntry(StandardEntryType.Misc)
+            .withField(StandardField.TITLE, "Effective Java")
+            .withField(StandardField.YEAR, "2018")
+            .withField(StandardField.SUBTITLE, "best practices für die Java-Plattform")
+            .withField(StandardField.AUTHOR, "Bloch, Joshua")
+            .withField(StandardField.TITLEADDON, "Joshua Bloch")
+            .withField(StandardField.EDITION, "3. Auflage, Übersetzung der englischsprachigen 3. Originalausgabe 2018")
+            .withField(StandardField.FILE, "ParsedFileField{description='', link='http://search.ebscohost.com/login.aspx?direct=true&scope=site&db=nlebk&db=nlabk&AN=1906353', fileType='PDF'}")
```

Comment:
Better use withFiles(List.of(new LInkedFile(

Reply:
Was a bug in the fetcher. Fixed at https://github.com/JabRef/jabref/pull/11852/commits/654616cd448db90e0bbae13c632726d5b77fa870.

Reply:
One more: https://github.com/JabRef/jabref/pull/11852/commits/d1c33d93d8c32c646e3cca41ab17cd07518f2b0d